site/content/en/docs/Reference/gameserverallocation.md

```diff
@@ -25,7 +25,8 @@ spec:
       game: my-game
     matchExpressions:
       - {key: tier, operator: In, values: [cache]}
-  # ordered list of preferred allocations out of the `required` set.
+  # An ordered list of preferred GameServer label selectors
+  # that are optional to be fulfilled, but will be searched before the `required` selector.
```

Comment:
"If any of the Preferred selectors are matched, the required selector is not considered" might be in order here.